### PR TITLE
2001 add application and process json files in zip

### PR DESCRIPTION
**Pull request by constantin-ciobotaru**

### CODE REVIEW

File: activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ModelType.java [hunk 1]
```diff
@@ -24,4 +24,6 @@
     String getName();
 
     String getFolderName();
+
+    String getContentFileExtension();
```
Comment (erdemedeiros) [inline activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ModelType.java:28]: I'm wondering processes are not going to be a problem as there are two possible extensions `bpmn20.xml` and `**.bpmn`.

File: activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ProcessModelType.java [hunk 2]
```diff
@@ -28,6 +30,8 @@
 
     public static final String PROCESSES = "processes";
 
+    public static final String BPMN20_XML = "bpmn20.xml";
```
Comment (erdemedeiros) [inline activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ProcessModelType.java:31]: As my previous comment `**.bpmn` is also a valid extension for processes (maybe legacy only, I'm not sure).

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/java/org/activiti/cloud/services/organization/rest/controller/ApplicationControllerIT.java [hunk 3]
```diff
@@ -325,13 +331,33 @@ public void testExportApplication() throws Exception {
...
                 .hasName("application-with-models.zip")
                 .hasEntries(
+                        "application-with-models.json",
                         "processes/",
                         "processes/process-model-1.bpmn20.xml",
-                        "processes/process-model-2.bpmn20.xml")
+                        "processes/process-model-1.json",
+                        "processes/process-model-2.bpmn20.xml",
+                        "processes/process-model-2.json")
+                .hasJsonContentSatisfying("application-with-models.json",
```
Comment (erdemedeiros) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/java/org/activiti/cloud/services/organization/rest/controller/ApplicationControllerIT.java:338]: @constantin-ciobotaru just to let you know, for json assertions we have been using [JsonUnit](https://github.com/lukas-krecan/JsonUnit#fluent) with which you can write fluent assertions in similar way to AssertJ.

**Review by erdemedeiros — COMMENTED** [review]
@constantin-ciobotaru have some question before merging. Please, take a look on my comments.
Also, we definitely need to find a better solution than committing zip files.

File: activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ProcessModelType.java (hunk 2 above)
Comment (constantin-ciobotaru) [reply activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ProcessModelType.java:31]: I've added allowedFileExtensions as string array of "bpmn20.xml", "bpmn.xml" and "bpmn". I hope, this covers all possibilities.

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/java/org/activiti/cloud/services/organization/rest/controller/ApplicationControllerIT.java (hunk 3 above)
Comment (constantin-ciobotaru) [reply activiti-cloud-services-org/activiti-cloud-services-org-rest/src/test/java/org/activiti/cloud/services/organization/rest/controller/ApplicationControllerIT.java:338]: Moved to json-unit-fluent. Thanks.

**Pull request merged**